packaging/install

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
This file should be removed.

This is the script referred to previously.

Reply:
removed